iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeRequestOtpView.swift

```diff
@@ -0,0 +1,202 @@
+            
+            HStack { Spacer() }
+            
+            ZStack {
+                
+                RoundedRectangle(cornerRadius: 10)
+                    .fill(.gray.opacity(0.25))
+                    .frame(width: 300, height: 450)
+                    .padding(.top, 0)
+                
+                VStack {
+                    Image(.msdkPhone)
+                        .resizable()
+                        .blur(radius: 0.0)
+                        .frame(width: 150, height: 150)
+                        .padding(.bottom, 50)
+                        .shadow(color: .black, radius: 3)
+                    
+                    if isAuthenticating {
+                        ProgressView()
+                    }
+                    
+                    if !errorMessage.isEmpty {
+                        Text(errorMessage)
+                            .frame(width: 250, height: 50)
+                            .foregroundStyle(.red)
+                    } else {
+                        Spacer().frame(width: 250, height: 65)
+                    }
+                    
+                    TextField("Username", text: $username)
+                        .autocapitalization(.none)
+                        .buttonStyle(.borderless)
+                        .disableAutocorrection(true)
+                        .foregroundColor(.blue)
+                        .frame(maxWidth: 250)
+                        .multilineTextAlignment(.center)
+                        .padding(.top, 25)
+                        .zIndex(2.0)
+                    
+                    Picker(
+                        "OTP Verification Method",
+                        selection: $otpVerificationMethod) {
+                            Text("Email").tag(OtpVerificationMethod.email)
+                            Text("SMS").tag(OtpVerificationMethod.sms)
+                        }
+                        .frame(maxWidth: 250)
+                        .zIndex(2.0)
+                    
+                    Button {
+                        onRequestOtpTapped()
+                    } label: {
+                        HStack {
+                            Image(systemName: "key.radiowaves.forward")
+                            Text("Request One-Time-Passcode")
+                        }.frame(minWidth: 150)
+                    }
+                    .buttonStyle(.bordered)
+                    .tint(.blue)
+                    .zIndex(2.0)
+                }
+            }
+            
+            Spacer()
+        }
+        .background(
+            Gradient(colors: [.blue, .cyan, .green]).opacity(0.6)
+        )
+        .blur(radius: isAuthenticating ? 2.0 : 0.0)
+    }
+    
+    ///
+    /// Submits the OTP delivery request when the request button is tapped.  This submits a request to the
+    /// `init/passwordless/login`endpoint and opens the submit OTP verification view on success.
+    ///
+    private func onRequestOtpTapped() {
```

Comment:
Everything in this view is straight-forward SwiftUI and this is the key logic using MSDK native login manger.